### ISSUE TITLE
Fine-tuning results

### ISSUE THREAD

**Issue by kimdanni** [shermanlian/IC-ViT#6]
<img width="744" height="237" alt="Image" src="https://github.com/user-attachments/assets/75bff1d0-b2b3-410b-95ef-41aa93958a19" />

First of all, thank you for sharing the JumpCP pre-trained models and your hard work on this project.

I am currently trying to fine-tune the jumpcp_pretrain_checkpoint.pth on Jumpcp dataset. However, I am having trouble reproducing the reported performance (around 80%).

Currently, my best test accuracy is plateauing at approximately 65%, which is very similar to the results I get when training from scratch. I suspect I might be missing a specific configuration or a crucial preprocessing step.

Could you please provide some guidance on the following?

Below is the script I am currently using for the fine-tuning process:
python icvit/main/main_supervised.py \
    trainer.devices=2 \
    trainer.max_epochs=100 \
    meta_arch/backbone=icvit_small \
    meta_arch.backbone.args.in_chans=8 \
    meta_arch.backbone.args.patch_size=16 \
    meta_arch.backbone.checkpoint=../dino-imagenet/checkpoints/jumpcp_pretrain_checkpoint.pth\
    meta_arch.target='label' \
    meta_arch.num_classes=161 \
    data@train_data=jumpcp \
    data@val_data_dict=[jumpcp_val_fl,jumpcp_test_fl] \
    train_data.jumpcp.loader.batch_size=32 \
    transformations@train_transformations=cell \
    transformations@val_transformations=cell

**Comment by shermanlian** [comment]
Hi, it looks like you are running the ”partial“ scenario (FL only).
For the full evaluation, please set ”data.val_data_dict = [jumpcp_val, jumpcp_test]“. Let me know if it still doesn't work!
